brave/src/main/java/brave/handler/FinishedSpanHandler.java

```diff
@@ -64,9 +65,60 @@
    * visible to later handlers, including Zipkin.
    * @return true retains the span, and should almost always be used. false drops the span, making
    * it invisible to later handlers such as Zipkin.
+   *
+   * @see #supportsOrphans() If you are scrubbing personal information, consider supporting orphans.
    */
   public abstract boolean handle(TraceContext context, MutableSpan span);
 
+  /**
+   * Normally, {@link #handle(TraceContext, MutableSpan)} is only called upon explicit termination
+   * of a span: {@link Span#finish()}, {@link Span#finish(long)} or {@link Span#flush()}. When this
+   * method returns true, the callback will also receive data orphaned due to spans being never
+   * terminated or data added after termination. It is important to understand this, especially if
+   * your handler is performing work like redaction. This sort of work needs to happen on all data,
+   * not just the success paths.
+   *
+   * <p><h3>What is an orphaned span?</h3>
+   * Brave adds an {@link Span#annotate(String) annotation} "brave.flush" when data remains
+   * associated with a span when it is garbage collected. This is almost always a bug. For example,
+   * calling {@link Span#tag(String, String)} after calling {@link Span#finish()}, or calling {@link
+   * Tracer#nextSpan()} yet never using the result. To track down bugs like this, set the logger
+   * {@link PendingSpans} to FINE level.
+   *
+   * <p><h3>Why handle orphans?</h3>
+   * Use cases for handling orphans include redaction, trimming the "brave.flush" annotation,
+   * logging a different way than default, or incrementing bug counters. For example, you could use
+   * the same SSN cleaner here as you do on the success path.
```

Comment:
Think SSN is social security number but it's a pretty unfamiliar term for most I think. credit card number or password might be more global.

Reply:
ack better idea. will change